include/mp++/real128.hpp

```diff
@@ -1514,8 +1634,150 @@ inline real128 cos(real128 x)
     return x.cos();
 }
 
+/// Unary tangent.
+/**
+ * @param x the \link mppp::real128 real128\endlink whose tangent will be computed.
+ *
+ * @return the tangent of \p x.
+ */
+inline real128 tan(real128 x)
+{
+    return x.tan();
+}
+
+/// Unary inverse sine.
+/**
+ * @param x the \link mppp::real128 real128\endlink whose inverse sine will be computed.
+ *
+ * @return the inverse sine of \p x.
+ */
+inline real128 asin(real128 x)
+{
+    return x.asin();
+}
+
+/// Unary inverse cosine.
+/**
+ * @param x the \link mppp::real128 real128\endlink whose inverse cosine will be computed.
+ *
+ * @return the inverse cosine of \p x.
+ */
+inline real128 acos(real128 x)
+{
+    return x.acos();
+}
+
+/// Unary inverse tangent.
+/**
+ * @param x the \link mppp::real128 real128\endlink whose inverse tangent will be computed.
+ *
+ * @return the inverse tangent of \p x.
+ */
+inline real128 atan(real128 x)
+{
+    return x.atan();
+}
+
+/** @} */
+
+/** @defgroup real128_trig real128_trig
+ *  @{
+ */
+
+/// Unary hyperbolic sine.
+/**
+ * @param x the \link mppp::real128 real128\endlink whose hyperbolic sine will be computed.
+ *
+ * @return the hyperbolic sine of \p x.
+ */
+inline real128 sinh(real128 x)
+{
+    return x.sinh();
+}
+
+/// Unary hyperbolic cosine.
+/**
+ * @param x the \link mppp::real128 real128\endlink whose hyperbolic cosine will be computed.
+ *
+ * @return the hyperbolic cosine of \p x.
+ */
+inline real128 cosh(real128 x)
+{
+    return x.cosh();
+}
+
+/// Unary hyperbolic tangent.
+/**
+ * @param x the \link mppp::real128 real128\endlink whose hyperbolic tangent will be computed.
+ *
+ * @return the hyperbolic tangent of \p x.
+ */
+inline real128 tanh(real128 x)
+{
+    return x.tanh();
+}
+
+/// Unary inverse hyperbolic sine.
+/**
+ * @param x the \link mppp::real128 real128\endlink whose inverse hyperbolic sine will be computed.
+ *
+ * @return the inverse hyperbolic sine of \p x.
+ */
+inline real128 asinh(real128 x)
+{
+    return x.asinh();
+}
+
+/// Unary inverse hyperbolic cosine.
+/**
+ * @param x the \link mppp::real128 real128\endlink whose inverse hyperbolic cosine will be computed.
+ *
+ * @return the inverse hyperbolic cosine of \p x.
+ */
+inline real128 acosh(real128 x)
+{
+    return x.acosh();
+}
+
+/// Unary inverse hyperbolic tangent.
+/**
+ * @param x the \link mppp::real128 real128\endlink whose inverse hyperbolic tangent will be computed.
+ *
+ * @return the inverse hyperbolic tangent of \p x.
+ */
+inline real128 atanh(real128 x)
+{
+    return x.atanh();
+}
+
 /** @} */
 
+/** @defgroup real128_miscfuncts real128_miscfuncts
+ *  @{
+ */
+/// Error function.
+/**
+ * @param x the \link mppp::real128 real128\endlink whose erf will be computed.
+ *
+ * @return the erf of \p x.
+ */
+inline real128 erf(real128 x)
+{
+    return x.erf();
+}
+
+/// Natural logarithm of the gamma funcion
```

Comment:
In the real class, gamma functions and related are grouped into their own group. E.g., see:

https://bluescarni.github.io/mppp/real.html#gamma-functions

Could you do the same with real128's lgamma function?

Reply:
I noted you use gamma and not tgamma as a name for the gamma function (as opposed to whats in quadmath and std). Is there a reason?